circle.yml

```diff
@@ -32,6 +33,7 @@ notify:
 test:
   override:
     - bundle exec rake lint:bash
+    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && gometalinter.v1 --deadline=20s cmd/
```

Comment:
@Bowbaq do you have any suggestions on which linters are a good choice, and which ones are slow or overly pedantic? Right now the linters take way too long to run as part of the dev cycle, up to 15 seconds!